single_tech_samples/databricks/sample2_enterprise_azure_databricks_environment/README.md

```diff
@@ -0,0 +1,273 @@
+  
+**Hub Vnet**
+![alt text](../Common_Assets/Images/ADE_Hub.png "Logo Title Text 1")
+
+**Spoke Vnet**
+![alt text](../Common_Assets/Images/ADE_Spoke.png "Logo Title Text 1")
+
+5.Azure Firewall with rules configured
+
+**Network Rules**
+![alt text](../Common_Assets/Images/ADE_Firewall.png "Logo Title Text 1")
+
+**Application Rules**
+![alt text](../Common_Assets/Images/ADE_Firewall_AR.png "Logo Title Text 1")
+
+Please note that the firewall rules configured in this sample are as per Microsoft Azure documentation captured [here](https://docs.microsoft.com/en-us/azure/databricks/administration-guide/cloud-configurations/azure/udr). The routes are configured for West US region. In case you need to deploy this sample in any other Azure region, please alter the [ARM template for the firewall](firewall/firewall.template.json) with the IP addresses for the target region.
+
+6.Azure public IP address associated with the firewall
+
+![alt text](../Common_Assets/Images/ADE_IP.png "Logo Title Text 1")
+
+7.Azure Routing table with routes configured
+
+**Subnets**
+![alt text](../Common_Assets/Images/ADE_Subnet.png "Logo Title Text 1")
+
+**Routes**
+![alt text](../Common_Assets/Images/ADE_Route.png "Logo Title Text 1")
+
+8.Azure Network Security Group
+
+![alt text](../Common_Assets/Images/ADE_NSG.png "Logo Title Text 1")
+
+8.Azure Private Link
+
+> **NOTE:** Configuring Private links require provisioning of Network Interface cards and Private DNS zones. The following screenshot illustrates the resources configured for two private links. One for the Storage and Another for Key vault.
+>
+![alt text](../Common_Assets/Images/ADE_PL.png "Logo Title Text 1")
+
+### 3.4. Deployment validation
+
+The following steps can be performed to validate the correct deployment of this sample:
+
+1. Users with appropriate access rights should be able to:
+
+   1. Launch the workspace from the Azure portal.
+   2. Access the control plane for the storage account and key vault through the Azure portal.
+   3. View the secrets configured in the Azure Key vault.
+   4. View deployment logs in the Azure resource group
+   ![alt text](../Common_Assets/Images/ADE_Deployment_Logs.png "Logo Title Text 1").
+   5. Key vault and storage is not accessible outside Azure Databricks workspace.
+   6. Changing the firewall rules to deny traffic from the control plane will prevent the Azure Databricks cluster from functioning.
+   ![alt text](../Common_Assets/Images/ADE_Firewall_Deny.png "Logo Title Text 1").
+
+### 3.5. Clean-up
+
+The clean-up script can be executed to clean up the resources provisioned in this sample. Following are the steps to execute the script:
+
+1. Navigate to (CD) `single_tech_samples/databricks/sample1_basic_azure_databricks_environment/`.
+
+2. Run '/destroy.sh'
+
+The following screenshot displays the log for a successful clean-up run:
+
+  ![alt text](../Common_Assets/Images/ADE_Script_Teardown.png "Logo Title Text 1")
```

Comment:
Add a "Next steps" section pointing to Sample 3.

Reply:
@namitms Addressing

Reply:
@devlace Resolved